lib/cli.ex

```diff
@@ -19,11 +23,14 @@ defmodule Onigumo.CLI do
     components = Enum.join(Map.keys(@components), ", ")
 
     IO.puts("""
-    Usage: onigumo [COMPONENT]
+    Usage: onigumo [OPTION]... [COMPONENT]
 
     Simple program that retrieves HTTP web content as structured data.
 
     COMPONENT\tOnigumo component to run, available: #{components}
+
+    OPTIONS:
+    -C, --working-dir=DIR\tChange to <DIR> before processing remaining files
```

Comment:
Change _what_? What about “change the (current) working directory”. I know that tar uses your wording, but tar is generally confusing. 🤯

What do you mean by “before processing remaining files”? This is exactly what tar says, but I am not sure it’s appropriate in our context. Even though Onigumo in fact processes files, just as any software does, I wouldn’t call it that way if I was talking about it. Moreover, the “remaining” there sounds confusing too: it would make more sense in case of resumed run, but, again, it’s not the primary use case. Let’s say something more universal and straightforward like Meson’s “before running”.

```suggestion
    -C, --working-dir=DIR\tChange the working directory before running
```

Reply:
Alright that is good point I've changed to running.